### ISSUE TITLE
[AnalyticalTable]: [a11y. Screen readers don't read whole information]

### ISSUE THREAD

**Issue by Kalita-Roman** [SAP/ui5-webcomponents-react#7410]
### Describe the bug

When the AnalyticalTable enters any pop-in mode and some cell content is moved into the first column, not all the content is read by screen readers.

Violation of:
[WCAG 2.1, 4.1.2, Level A, Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
[EN 301 549 V3.2.1 (2021-03)](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf) 9.4.1.2, 10.4.1.2, and 11.4.1.2

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/iframe.html?globals=&args=&id=data-display-analyticaltable--responsive-columns&viewMode=story

### Reproduction steps

1. Open an `AnalyticalTable` with any `popinDisplay` mode, e.g. https://sap.github.io/ui5-webcomponents-react/v2/iframe.html?globals=&args=&id=data-display-analyticaltable--responsive-columns&viewMode=story
2. Shrink the viewport to achieve a pop-in effect.
3. Run a screen reader (NVDA or JAWS)
4. Try to read out all info in the cells

Current behaviour:
![Image](https://github.com/user-attachments/assets/b1ee5396-b00c-4abf-b147-9e20b107ff8e)
JAWS reads:

> Current width: auto
> grid with 4 columns and 8 of 500 rows
> Name column header
> column 1 row 1
> Name Allen Best Contains Controls
> Row 2
> Name Combs Fleming Contains Controls
> Row 3

### Expected Behaviour

On accessing a data cell in the ‘Name’ (1st) column, the screen reader must announce all the content within the cell, along with any necessary additional information, without omitting anything.

### UI5 Web Components for React Version

v2.10.1

### UI5 Web Components Version

v2.10.1

### Browser

Chrome

### Additional Context

This component behaviour isn't compliant with [European accessibility act](https://commission.europa.eu/strategy-and-policy/policies/justice-and-fundamental-rights/disability/union-equality-strategy-rights-persons-disabilities-2021-2030/european-accessibility-act_en)

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
